Pr reorder classes

borrob opened the pull request:
See if you like this one or not. I moved some of the classes to a (I think) more appropriate place.

narcode commented:
Thanks Rob! I agree with all your open issues/suggestions. I will get to it once i get back from Holidays!

borrob committed:
Make scripts self executable

Perhaps the users still need to ``chmod +x hello_world.py``, but the shebang
operater is there!

borrob committed:
Merge branch 'pr_executing_scripts' into develop

narcode committed:
start mottiptes dev

narcode committed:
python motippets first commit and wingIDE test

narcode committed:
mapping for motippets

narcode committed:
motippets migration to python

narcode committed:
added mapping fro Motippets

narcode committed:
refactor main Motippets class and added interface for motifs

narcode committed:
readme update

narcode committed:
readme update and test git with new IDE

borrob committed:
Merge remote-tracking branch 'upstream/develop' into develop

borrob committed:
Update style

borrob committed:
Move colors to the Setup file.

The colors aren't only meant for the instructions. They are much more general
and therefore they fit better in setup/init kind of thing.

borrob committed:
Move mottipets motifs

The motifs are similar to 'Mappings'. Multiple versions/settings can co-exists
next to each other no problem.

narcode commented:
I agree with the new places for the classes!
